types/passport-google-oauth2/index.d.ts

```diff
@@ -43,7 +59,7 @@ export class Strategy implements Strategy {
   name: string;
   authenticate: (req: Request, options?: object) => void;
 
-  constructor(options: StrategyOptionsWithRequest, verify: VerifyFunctionWithRequest);
+  constructor(options: StrategyOptionsWithRequest, verify: VerifyFunctionWithRequest | VerifyFunctionWithRequestAndParams);
```

Comment:
I think it might be a better idea to introduce a new constructor for this so users don't need to add type guards using these signatures. What do you think?

Reply:
I did do this originally but the linter failed told me to do it like this

Reply:
Oh so I see. I've gone back and had a look at the linter rule, I'm not entirely sure it's a great rule but that's not up to me! Well with that in mind I'll approve this 😄